fog/sample-paykit/src/cached_tx_data/mod.rs

```diff
@@ -369,6 +393,48 @@ impl CachedTxData {
             log::error!(self.logger, "Fog view protocol error: {}", err);
         }
 
+        log::trace!(
```

Comment:
Nit: I would upgrade this to debug, it is not expected to happen often and won't be noisy.